examples/runCfhtQuickTest.sh

```diff
@@ -7,12 +7,45 @@ CAMERA=CfhtQuick
 CONFIG_FILE="${PRODUCT_DIR}/config/cfhtConfig.py"
 MAPPER=lsst.obs.cfht.MegacamMapper
 
-"${PRODUCT_DIR}/examples/processData.sh" \
-    -c "$CAMERA" \
-    -m "$MAPPER" \
-    -v "$VALIDATION_DATA_DIR" \
-    -f "$CONFIG_FILE" \
-    -- "$@"
-"${PRODUCT_DIR}/examples/validateRepo.sh" \
-    -c "$CAMERA" \
-    -- "$@"
+print_error() {
+    >&2 echo "$@"
+}
+
+DOPROCESS=true
+DOVERIFY=true
+
+usage() {
+    print_error
+    print_error "Usage: $0 [-pv] [-h] [-- <extra options to validateDrp.py>]"
+    print_error
+    print_error "Specifc options:"
+    print_error "   -p          Skip processing?"
+    print_error "   -v          Skip verification?"
+    print_error "   -h          show this message"
+    exit 1
+}
+
+# thank OSX for not including getopt
+while getopts "vp" option; do
+    case "$option" in
+        p)  DOPROCESS=false;;
+        v)  DOVERIFY=false;;
+        h)  usage;;
+        *)  usage;;
```

Comment:
Technically, `h` and `*` don't need to be separate cases as `*` will match `h` but that might be a slightly non-obvious to future maintainers.  However, it should be possible to condense these two matches with `h | *)`.

Reply:
I'm removing the `*` as we discussed.